openupgrade_scripts/scripts/event/17.0.1.8/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,67 @@
+---Models in module 'event'---
+---Fields in module 'event'---
+event        / event.event              / _order                        : _order is now 'date_begin, id' ('date_begin')
+event        / event.event              / activity_user_id (many2one)   : not related anymore
+event        / event.event              / activity_user_id (many2one)   : now a function
+# NOTHING TO DO
+event        / event.event              / auto_confirm (boolean)        : DEL
```

Comment:
As this is plain text, you need to leave a blank line between each block where you insert comments, or it's very difficult to be read.